test/utils/ConstraintsSpec.scala

```diff
@@ -49,8 +49,8 @@ class ConstraintsSpec extends SpecBase with Constraints {
         result mustBe Valid
       }
 
-      "return invalid result is years are within 6" in new Setup {
-        def monthOld: LocalDate = LocalDateTime.now().minusMonths(8).toLocalDate
+      "return invalid result if request is after current month" in new Setup {
```

Comment:
We should not change the test description (as it is written for some functionality) unless it is not valid anymore (after change in the code (for relevant functionality)). If its failing (and not dependent on your lib change) then either there is an issue in the code or the way test is written. If its taking longer to fix this, then ignore the test (by replacing in to ignore) and add a //TODO: <comment_description> . Provide description like Need to looked at as its failing for some dates

Reply:
As advised- Have restored and ignored the earlier failing test kept the new test